e2e/resource_quota_exceeded_test.go

```diff
@@ -166,63 +165,41 @@ var _ = Describe("exceeding Tenant resource quota", func() {
 				}
 				_, err = cs.AppsV1().Deployments(namespace).Create(context.TODO(), d, metav1.CreateOptions{})
 				return
-			}, 15*time.Second, time.Second).Should(Succeed())
+			}, defaultTimeoutInterval, defaultPollInterval).Should(Succeed())
 		}
 		for _, ns := range nsl {
 			n := fmt.Sprintf("capsule-%s-1", tnt.GetName())
 			rq := &corev1.ResourceQuota{}
 			By("retrieving the Resource Quota", func() {
 				Eventually(func() error {
 					return k8sClient.Get(context.TODO(), types.NamespacedName{Name: n, Namespace: ns}, rq)
-				}, 15*time.Second, time.Second).Should(Succeed())
+				}, defaultTimeoutInterval, defaultPollInterval).Should(Succeed())
 			})
 			By("ensuring the status has been blocked with actual usage", func() {
 				Eventually(func() corev1.ResourceList {
 					_ = k8sClient.Get(context.TODO(), types.NamespacedName{Name: n, Namespace: ns}, rq)
 					return rq.Status.Hard
-				}, 15*time.Second, time.Second).Should(Equal(rq.Status.Used))
+				}, defaultTimeoutInterval, defaultPollInterval).Should(Equal(rq.Status.Used))
 			})
 			By("creating an exceeded Pod", func() {
-				d := &v1.Deployment{
+				pod := &corev1.Pod{
```

Comment:
Rather than creating Deployment, checking ReplicaSet and finally conditions, is it more suitable to run a single Pod!